gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -55,33 +75,52 @@ gst_tensor_query_serversink_class_init (GstTensorQueryServerSinkClass * klass)
   gobject_class->get_property = gst_tensor_query_serversink_get_property;
   gobject_class->finalize = gst_tensor_query_serversink_finalize;
 
-  /** install property goes here */
+  g_object_class_install_property (gobject_class, PROP_HOST,
+      g_param_spec_string ("host", "Host", "The hostname to listen as",
+          DEFAULT_HOST, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_PORT,
+      g_param_spec_int ("port", "Port",
+          "The port to listen to (0=random available port)", 0,
+          65535, DEFAULT_PORT_SINK,
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_PROTOCOL,
+      g_param_spec_int ("protocol", "Protocol",
+          "The network protocol to establish connection", 0,
+          _TENSOR_QUERY_PROTOCOL_END, DEFAULT_PROTOCOL,
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_TIMEOUT,
+      g_param_spec_int ("timeout", "Timeout",
```

Comment:
`port` and `timeout` is unsigned integer.

Reply:
Fixed!